### PR TITLE
Revert "Revert "Minify prod css (#1386)" (#1392)"

### PR DESCRIPTION
**Pull request by ndmckinley**
This reverts commit a2d9bfe95a788daef5255a6ee5d05972fc574e6c.

### Description
This solution is only okay.  Because we have copied every class - including very many that we do not use - into `Styles.java`, and because we programmatically create utility classes, we have to keep an awful lot of CSS that we don't actually need.  Still, this brings us from 4.6 MB to 1.6 MB, which is something - and eliminates the prod/dev distinction, so we can see if we're about to screw up the styles in prod.

It also adds a tool to refresh styles without restarting `bin/run-dev`, which is a little helpful for development here.

We can get down to below a megabyte by removing all the styles from `Styles.java` that we do not use.  What we cannot do is analyze that file at minification-time to see which variables are in use.  I'll leave that choice - whether to delete all the classes we don't need - to someone else.

### Checklist
- [ ] Created tests which fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

### Issue(s)

### CODE REVIEW

File: bin/refresh-styles [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#! /bin/bash
+pushd $(git rev-parse --show-toplevel)
+
+# allocate a tty for better test output even though not strictly needed.
+docker run -it --rm -v /var/run/docker.sock:/var/run/docker.sock -v $(pwd)/universal-application-tool-0.0.1:/usr/src/universal-application-tool-0.0.1 --entrypoint npx civiform tailwindcss build -i ./app/assets/stylesheets/styles.css -o ./public/stylesheets/tailwind.css
```
Comment (bion) [inline bin/refresh-styles:5]: suggestion (non-blocking): break onto multiple lines for easier readability

File: universal-application-tool-0.0.1/app/views/style/BaseStyles.java [hunk 2]
```diff
@@ -1,5 +1,12 @@
 package views.style;
 
+/**
+ * Constant class containing the names of styles that we have added to tailwind.
+ *
+ * <p>This file is special - strings in this file, within double quotes, are *not* stripped from the
+ * tailwind CSS during production optimization. If you add a string here, run bin/refresh-styles or
+ * restart bin/run-dev.
+ */
```
Comment (bion) [inline universal-application-tool-0.0.1/app/views/style/BaseStyles.java:9]: 👍🏻

File: bin/refresh-styles [hunk 3]
```diff
@@ -0,0 +1,7 @@
+#! /bin/bash
+pushd $(git rev-parse --show-toplevel)
+
+# allocate a tty for better test output even though not strictly needed.
```
Comment (hmhsu) [inline bin/refresh-styles:4]: nit: remove `test output`

File: universal-application-tool-0.0.1/app/views/style/BaseStyles.java [hunk 4]
```diff
@@ -1,5 +1,12 @@
 package views.style;
 
+/**
+ * Constant class containing the names of styles that we have added to tailwind.
```
Comment (hmhsu) [inline universal-application-tool-0.0.1/app/views/style/BaseStyles.java:4]: praise: thanks for adding the docstring

**Review by bion — APPROVED** [review]

**Review by hmhsu — APPROVED** [review]

**Pull request merged**